### PR TITLE
Docs and testing for ticket-plugin-base

### PR DESCRIPTION
**Pull request by hmpf**

### CODE REVIEW

File: src/argus/incident/ticket/dummy.py [hunk 1]
```diff
@@ -1,5 +1,15 @@
+"""
+Minimal ticket creation plugin
+
+Serves as an example and is also used for testing.
+
+Inspired by django.core.mail.outbox, see https://docs.djangoproject.com/en/3.2/topics/testing/tools/#email-services
+django.core
```
Comment (johannaengland) [inline src/argus/incident/ticket/dummy.py]: Is this line supposed to be here? Or just a copy paste error?

File: src/argus/incident/ticket/dummy.py [hunk 2]
```diff
@@ -14,45 +24,78 @@
...
     created_tickets = []
 
 
 class TicketTestClient:
+    "Mock ticket client, for testing"
+
     def __init__(self, endpoint, authentication):
+        """Initialize the client that will chat to the ticket system
+
+        All ticket systems we have seen so far seems to use this pattern to
```
Comment (johannaengland) [inline src/argus/incident/ticket/dummy.py]:
```suggestion
        All ticket systems we have seen so far seem to use this pattern to
```

File: tests/incident/ticket_plugins/test_dummy.py [hunk 3]
```diff
@@ -25,5 +25,5 @@ def test_create_ticket_writes_to_local_variable(self):
 
         url = dummy_class.create_ticket(incident)
 
-        self.assertEqual(url, "www.example.com")
+        self.assertTrue(bool(url.strip()), 'Return url can never be empty')
```
Comment (johannaengland) [inline tests/incident/ticket_plugins/test_dummy.py]: We could here also test the url using `django.core.validators.URLValidator`?

**Review by johannaengland — CHANGES_REQUESTED** [review]

File: src/argus/incident/ticket/dummy.py (hunk 1 above)
Comment (hmpf) [reply src/argus/incident/ticket/dummy.py]: You've marked what looks like 4 lines. I can only see the top three lines in this browser. Do you mean what is on line 7?

File: tests/incident/ticket_plugins/test_dummy.py (hunk 3 above)
Comment (hmpf) [reply tests/incident/ticket_plugins/test_dummy.py]: Yep. I switched to a generated fake url to make it harder for us. There are in fact systems that use different domain names for reading and writing. (Well I hope they aren't too common.)

Comment (hmpf) [reply tests/incident/ticket_plugins/test_dummy.py]: I have added  url-validation.

File: src/argus/incident/ticket/dummy.py (hunk 1 above)
Comment (johannaengland) [reply src/argus/incident/ticket/dummy.py]: Just line 7

Comment (hmpf) [reply src/argus/incident/ticket/dummy.py]: Thx. Yeah, line 7 was a copy paste error.

**Review by johannaengland — APPROVED** [review]

**Pull request merged**